src/electionguard-ui/ElectionGuard.UI.Lib/Models/KeyCeremonyRecord.cs

```diff
@@ -5,9 +5,9 @@ namespace ElectionGuard.UI.Lib.Models;
 /// <summary>
 /// Data for the key ceremony that is saved to the database
 /// </summary>
-public partial class KeyCeremony : DatabaseRecord
+public partial class KeyCeremonyRecord : DatabaseRecord

```

Comment:
You will also need to change the UI code to use this new class name.  This is making the UI fail to build.